README.md

```diff
@@ -60,9 +60,9 @@ To get a static version (that can work without t-rex and postgres):
 
 ### Style Edition
 
-The background style is in `web/glstyle.json` while the foreground style is in `web/vapour-style.json`. Both can be edited manually or with style editor.
+The displayed map used bus vector tiles served by the t-rex server and [Jawg vector tiles](https://jawg.io) with a custom theme for the background style.
 
-With the t-rex tiles server running locally you can upload `vapour-style.json` to the online [Maputnik Editor](http://editor.openmaptiles.org) and exported back.
+While the t-rex tiles server is running you can upload `glstyle.json` to the online [Maputnik Editor](http://editor.openmaptiles.org), make your changes and export it back to the project.
 
```

Comment:
As there is two sources of tilesset I suggest to not merge the two styles.
It's really more easy to deals in the editor only with the relevant part, and so the background can be easy replaced.

The mix of the interlacement of layers isn't required, it is?

Reply:
it is needed indeed, to put the layers in a consistent order and enhance lisibility.
For instance, the name of the street are now written on top of the streets, even if there is a bus track in the street.

We should provide an unified vision of the bus data in their context: separate the displayed info by source is a technical point of view but it does not make sense for the final user of the map.

Reply:
Ok. In this case the two styles can be blended with javascript, like previously done.
https://www.mapbox.com/mapbox-gl-js/example/geojson-layer-in-stack/

Reply:
This solution only have drawbacks :
* you won't be able to modify the style in Maputnik because it will be inside the javascript
* you won't be able to easily replace the background because the name of some ot its layers will need to be hardcoded in javascript

We need to mix the layers to create a consistent looking.

Reply:
* There is already two style files merged by javascrit, and the two styles are editable in Maputnik